### ISSUE TITLE
Scrollbar ignores scroll events while smoothly scrolling a single step

### ISSUE THREAD

**Issue by BMicraft** [xournalpp/xournalpp#2540]
**Affects versions :**
 - OS: Arch Linux (5.4.82-1-lts)
 - (Linux only) Desktop environment: KDE Plasma 5.20.4
 - Which version of libgtk do you use
      3.24.24
 - Version of Xournal++:
      1.1.0+dev
 - Installation method:
      Arch Linux PKGBUILD with commit 7ab24766206264c179d0a1e405cb2c1d43c7e83c

**Describe the bug**
When scrolling by hovering over the scroll bar and using the mouse wheel xournalpp will 'animate' the scroll action to be smooth, but while it does that all successive scroll events are dropped until the animation is complete.

**To Reproduce**
1. Scroll fast using mouse wheel while hovering over the scroll bar.
2. Notice how it ignores scroll events until the target of the first scroll event is reached. 

**Expected behavior**
The scroll bar should behave exactly like the document itself, whether that is with or without smooth scrolling. For now maybe disable smooth scrolling and bring it back when it works correctly?

**Comment by MichelReims** [comment]
Is there a common point with #2523 ?

**Comment by BMicraft** [comment]
> Is there a common point with #2523 ?
No, most likely not as this only concerns the scroll bar (and scrolling works even there, albeit slowly)

Your issue seems similar to #2138 (which is almost fixed). Have you tried disabling zoom to fit in the meantime?
You also said however that it works with the touchscreen workaround disabled, so it may not be similar after all.

**Comment by MichelReims** [comment]
I said  that it works with the touchscreen workaround disabled, but in previous version of 2020-12-04. Since, that no works with workaround enabled or disabled

**Comment by BMicraft** [comment]
> I said that it works with the touchscreen workaround disabled, but in previous version of 2020-12-04. Since, that not works with workaround enabled or disabled

Did you test if it is related to "Zoom to Fit"?

**Comment by MichelReims** [comment]
I have tested with several pages, with differents zoom

**Comment by MichelReims** [comment]
Have tou solved the problem ?

**Comment by BMicraft** [comment]
@MichelReims I've never encountered your issue

**Comment by MichelReims** [comment]
Solved Your problem ?

**Comment by BMicraft** [comment]
@MichelReims well no, considering this is a bug this won't be 'solved' without fixing the code, which is not something I can do myself

**Comment by MichelReims** [comment]
did you have this problem with previous version of libgtk ?

**Comment by BMicraft** [comment]
I don't know how I would go about testing this without breaking my arch install

**Comment by MichelReims** [comment]
Can you test with commit 891c972c27a335113f8a424ae7d3adcbc1ed3f18 ?

**Comment by MichelReims** [comment]
Then test with 925acf9eefce1775bb3b174d2742d620243a3412

**Comment by MichelReims** [comment]
how did you know you were using libgtk 3.24.24.  You marked it in the description of your problem

**Comment by BMicraft** [comment]
My package manager told me, but if you're on windows you can probably also be able to `xournalpp --version`
